src/main/java/net/pms/dlna/protocolinfo/DLNAOrgOperations.java

```diff
@@ -0,0 +1,342 @@
+/*
+ * Universal Media Server, for streaming any media to DLNA compatible renderers
+ * based on the http://www.ps3mediaserver.org. Copyright (C) 2012 UMS
+ * developers.
+ *
+ * This program is a free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; version 2 of the License only.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package net.pms.dlna.protocolinfo;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.fourthline.cling.support.model.Protocol;
+import net.pms.dlna.protocolinfo.ProtocolInfoAttributeName.KnownProtocolInfoAttributeName;
+import net.pms.util.ParseException;
+
+/**
+ * This class represents {@code DLNA.ORG_OP} attributes. This can be used for
+ * both DLNA and non-DLNA content.
+ *
+ * @author Nadahar
+ */
+public abstract class DLNAOrgOperations implements ProtocolInfoAttribute {
+
+	private static final long serialVersionUID = 1L;
+
+	/** The static attribute name always used for this class */
+	public static final ProtocolInfoAttributeName NAME = KnownProtocolInfoAttributeName.DLNA_ORG_OP;
+
+	/**
+	 * The static factory singleton instance used to retrieve
+	 * {@link DLNAOrgOperations} instances.
+	 */
+	public static final DLNAOrgOperationsFactory FACTORY = new DLNAOrgOperationsFactory();
+
+	/** The state */
+	protected final byte state;
+
+	/**
+	 * For internal use only, use {@link #FACTORY} to get instances.
+	 *
+	 * @param flagA flag A.
+	 * @param flagB flag B.
+	 */
+	protected DLNAOrgOperations(boolean flagA, boolean flagB) {
+		state = packFlags(flagA, flagB);
+	}
+
+	@Override
+	public ProtocolInfoAttributeName getName() {
+		return NAME;
+	}
+
+	@Override
+	public String getNameString() {
+		return NAME.getName();
+	}
+
+	@Override
+	public String getValue() {
+		return state > 0 ? ((state & 2) > 0 ? "1" : "0") + ((state & 1) > 0 ? "1" : "0") : "";
```

Comment:
If i'm not mistaken you authorize the 00 value to be throwed, but it's forbidden by the DLNA standard.
It say that the OP flag should be omit in that case.